x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/Transaction/ErrorCountBadge.tsx

```diff
@@ -13,20 +13,22 @@ import { Transaction } from '../../../../../typings/es_schemas/ui/Transaction';
 import { fontSize } from '../../../../style/variables';
 import { ErrorOverviewLink } from '../../../shared/Links/apm/ErrorOverviewLink';
 
-const LinkLabel = styled.span`
+const Title = styled.span`
   font-size: ${fontSize};
 `;
 
 interface Props {
   errorCount: number;
   transaction: Transaction;
   verbose?: boolean;
+  link?: boolean;
 }
 
 export const ErrorCountBadge: React.SFC<Props> = ({
   errorCount = 0,
   transaction,
-  verbose
+  verbose,
+  link = true
```

Comment:
This component (`ErrorCountBadge`) is only used in two places:

https://github.com/elastic/kibana/blob/70f03018f38138356274d47b30d7e7bdd170676e/x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/Transaction/StickyTransactionProperties.tsx:
```js
<ErrorCountBadge
  errorCount={errorCount}
  transaction={transaction}
  verbose
  link={false}
/>
```

and 

https://github.com/elastic/kibana/blob/70f03018f38138356274d47b30d7e7bdd170676e/x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/WaterfallItem.tsx#L202-L205

It doesn't look like they have anything else in common except for this:
```js
const errorCountBadge = (
  <EuiBadge color={euiThemeLight.euiColorDanger} {...linkProps}>
    {errorCount}
  </EuiBadge>
);
```

Probably better to simplify the component to just that and move the link-specific stuff into `waterfallItem.tsx` and the label into `StickyTransactionProperties.tsx`

Reply:
@sqren this should be fixed now, I think.